.github/workflows/release.yml

```
@@ -38,26 +38,26 @@ jobs:
         run: npm ci
         timeout-minutes: 2
 
-      - name: Exit pre-release mode and clean up
+      - name: Check pre-release mode
         run: |
-          echo "🔄 Ensuring we're not in pre-release mode for main branch..."
+          echo "🔍 Checking pre-release mode status..."
           
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)